analytic_base_department/views/analytic.xml

```diff
@@ -1,70 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-  <record id="view_account_analytic_account_form" model="ir.ui.view">
-    <field name="name">account.analytic.account.form</field>
-    <field name="model">account.analytic.account</field>
-    <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
-    <field name="arch" type="xml">
-      <field name="code" position="after">
-        <field name="department_id"/>
-      </field>
-    </field>
-  </record>
+    <record id="view_account_analytic_account_form" model="ir.ui.view">
+        <field name="name">account.analytic.account.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
+        <field name="arch" type="xml">
+            <field name="code" position="after">
+                <field name="department_id"/>
+            </field>
+        </field>
+    </record>
```

Comment:
No point in changing the XML indentation, but discussing that would be bikeshedding.